web/src/pages/admin/createTicket/index.js

```diff
@@ -131,6 +131,11 @@ export default function AdminUserTickets() {
                 icon={<IconBadge />}
                 {...register('cpf', {
                   required: 'Este campo é obrigatório',
+                  validate: (val) => {
+                    if (!val || val.length !== 11) {
+                      return 'Não é um CPF válido';
+                    }
```

Comment:
i think the mask cpf doesnt allow the users to digit more then 11 characters already so this part of the code will niver be triggered